src/extension/background-script/connectors/lnbits.ts

```diff
@@ -56,6 +56,11 @@ class LnBits implements Connector {
     });
   }
 
+  // not yet implemenetd
+  getInvoices() {
+    return new Error("Has not been implemneted on this connector: getInvoices");
```

Comment:
```suggestion
    console.error(`${this.constructor.name} does not implement the getInvoices call`);
    return new Error("Not yet supported with the currently used account.");
```

Was wondering if the error message should be more descriptive for the user?

Reply:
Good point. Added.